data/registry/instrumentation-js-node-honeycomb.yml

```
@@ -0,0 +1,12 @@
+title: Honeycomb.io OpenTelemetry Distro for Node.js
+registryType: instrumentation
+isThirdParty: true
+language: js
+tags:
+  - js
+  - Node.js
+  - instrumentation
+repo: https://github.com/honeycombio/honeycomb-opentelemetry-node
+license: Apache 2.0
+description: Honeycomb's distribution of OpenTelemetry for Node.js
+authors: Hound Technology Inc
```

Should this be the following (which is what is in the NOTICE files of those repos)?

```suggestion
authors: Honeycomb, Hound Technology, Inc
```